### DISCUSSION TITLE
Why no `__init__.py` in `tests/unit/**`?

### DISCUSSION THREAD

**Discussion by mattpap** [bokeh/bokeh#11321] [Internals and design (Q&A)] [answered]
I'm guessing this is intentional. This leads to relative imports like this:
```py
from _util_server import (
    http_get,
    url,
    websocket_open,
    ws_url,
)
```
which then can't be statically typed (well, at least not straight and would require `*.pyi` files or moving those utils under `bokeh._testing` or type checking test files individually).

/cc @bryevdv

**Comment by bryevdv** [comment, marked as answer]
See the discussion (especially towards the end of the section) here:

https://docs.pytest.org/en/6.2.x/goodpractices.html#tests-outside-application-code

TLDR; it is advised to use "src" directory layout of you add init files. I would like to move to src layout fwiw
